src/components/CreateAccountForm/CreateAccountForm.js

```diff
@@ -56,6 +57,31 @@ export default class CreateAccountForm extends Component {
     return true;
   }
 
+  getUserUpdateObject() {
+    // address should be included, even if null
+    let userUpdates = {address: this.getFullAddress()}
+      const inputsToCheck = [
+        this.state.firstName,
+        this.state.lastName,
+        this.state.email,
+        this.state.password,
+        this.state.zipCode
+      ]
+      const updateKeys = [
+        'firstName',
+        'lastName',
+        'email',
+        'password',
+        'zipCode'
+      ]
+      for (let i = 0; i < inputsToCheck.length; i++) {
+        let obj = {};
+        obj[updateKeys[i]] = inputsToCheck[i];
+        if (inputsToCheck[i]) Object.assign(userUpdates, obj);
```

Comment:
Is there a specific reason that we create an obj and use Object.assign(), instead of assigning the value directly? (eg: `userUpdates[updateKeys[i]] = inputsToCheck[i]`)

Btw I think you could shorten this part to the following snippet without even declaring `inputsToCheck`:
```
      for (const key of updateKeys) {
        if (this.state[key]) userUpdates[key] = this.state[key];
      }
```

Reply:
The main reason was to only add values that weren't null. Your way is more clear